ob_v2p0/index.md

```diff
@@ -629,7 +629,7 @@ If a property would be useful beyond a publisher's internal use, an [Extension](
 * <a id="dateTime"></a>DateTime - Open Badges must express timestamps as strings compatible with [ISO 8601](http://en.wikipedia.org/wiki/ISO_8601) guidelines, including the time and a time zone indicator. It is recommended to publish all timestamps in UTC. Previous versions of Open Badges allowed Unix timestamps as integers. Open Badges v2.0 requires string ISO 8601 values with time zone indicators. For example, `2016-12-31T23:59:59+00:00` is a valid ISO 8601 timestamp. It contains the year, month, day, `T` separator, hour number 0-23, minute, optional seconds and decimal microsecond, and a time zone indicator (+/- an offset from UTC or the `Z` designator for UTC).
 * URL - Fully qualified URL, including protocol, host, port if applicable, and path. Interpreters are only expected to interpret URLs in either the `http` or `https` schemes.
 * IRI - In JSON-LD and Linked Data, IRIs (Internationalized Resource Identifiers) may look like fully qualified URLs or be namespaced within the JSON-LD context to be expanded to a full IRI. The only known supported IRI schemes are `http` and `https`.
-* <a id="identityHash"></a>IdentityHash - A hash string preceded by a dollar sign ("$") and the algorithm used to generate the hash. The supported algorithms are [MD5](https://www.ietf.org/rfc/rfc1321.txt) and [SHA-256](http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf), identified by the strings `md5` and `sha256` respectively. For example: `sha256$28d50415252ab6c689a54413da15b083034b66e5` represents the result of calculating a SHA-256 hash on the string "mayze". For more information, see [how to hash & salt in various languages](https://github.com/mozilla/openbadges/wiki/How-to-hash-&-salt-in-various-languages.).
+* <a id="identityHash"></a>IdentityHash - A hash string containing an algorithm identifier and a resulting hash over the an identifier, separated by a dollar sign ("$") and the algorithm used to generate the hash. The only supported algorithms are [MD5](https://www.ietf.org/rfc/rfc1321.txt) and [SHA-256](http://nvlpubs.nist.gov/nistpubs/FIPS/NIST.FIPS.180-4.pdf), identified by the strings `md5` and `sha256` respectively. Resulting hash for each of these algorithms MUST be expressed in hexadecimal using uppercase (A-F, 0-9) or lowercase character (a-f, 0-9) sets. For example: `sha256$7a1a1b3f7d40552e4299180e346e3add12bf9a751a43d9c4ca4518febc2c60c6` represents the result of calculating a SHA-256 hash on the string "mayze". For more information, see [how to hash & salt in various languages](https://github.com/mozilla/openbadges/wiki/How-to-hash-&-salt-in-various-languages.).
```

Comment:
Is this meant to read "and a resulting hash of the identifier"?

Reply:
Fixed by https://github.com/IMSGlobal/openbadges-specification/pull/240/commits/c8b9b0e6ffa83281ac3069d5e3f83da8d8f7cd48